openpype/settings/entities/schemas/projects_schema/schema_project_traypublisher.json

```diff
@@ -78,6 +78,80 @@
                     }
                 ]
             }
+        },
+         {
+            "type": "dict",
+            "collapsible": true,
+            "key": "BatchMovCreator",
+            "label": "Batch Mov Creator",
+            "use_label_wrap": true,
+            "collapsible_key": true,
+            "children": [
+                {
```

Comment:
There are unused settings `family`. `identifier`, `label`, `icon`, `description` and `detailed_description`.